_i18n/en/documentation/general/app-permissions.md

```diff
@@ -0,0 +1,13 @@
+AntennaPod will not ask you for any general permissions. However you can give the following permissions:
```

Comment:
I don't really see why we would need to maintain this list. It just overloads users looking for help with the app by showing them stuff they don't need